.sync/Files.yml

```diff
@@ -27,83 +27,35 @@
 #       set up to use the YAML file. The file is simply available for pipelines to be connected
 #       if needed.
 group:
-# Azure Pipelines - Custom - microsoft/mu_feature_config
-  - files:
-    - source: .sync/azure_pipelines/custom/mu_feature_config/Ubuntu-GCC5.yml
-      dest: .azurepipelines/Ubuntu-GCC5.yml
-      template: true
-    - source: .sync/azure_pipelines/custom/mu_feature_config/Windows-VS.yml
-      dest: .azurepipelines/Windows-VS.yml
-      template: true
-    repos: |
-      microsoft/mu_feature_config
-
-# Azure Pipelines - Custom - microsoft/mu_feature_ipmi
-  - files:
-    - source: .sync/azure_pipelines/custom/mu_feature_ipmi/Ubuntu-GCC5.yml
-      dest: .azurepipelines/Ubuntu-GCC5.yml
-      template: true
-    - source: .sync/azure_pipelines/custom/mu_feature_ipmi/Windows-VS.yml
-      dest: .azurepipelines/Windows-VS.yml
-      template: true
-    repos: |
-      microsoft/mu_feature_ipmi
-
-# Azure Pipelines - Custom - microsoft/mu_feature_mm_supv
-  - files:
-    - source: .sync/azure_pipelines/custom/mu_feature_mm_supv/Ubuntu-GCC5.yml
-      dest: .azurepipelines/Ubuntu-GCC5.yml
-      template: true
-    - source: .sync/azure_pipelines/custom/mu_feature_mm_supv/Windows-VS.yml
-      dest: .azurepipelines/Windows-VS.yml
-      template: true
-    repos: |
-      microsoft/mu_feature_mm_supv
-
-# Azure Pipelines - Custom - microsoft/mu_oem_sample
-  - files:
-    - source: .sync/azure_pipelines/custom/mu_oem_sample/Ubuntu-GCC5.yml
-      dest: .azurepipelines/Ubuntu-GCC5.yml
-      template: true
-    - source: .sync/azure_pipelines/custom/mu_oem_sample/Windows-VS.yml
-      dest: .azurepipelines/Windows-VS.yml
-      template: true
-    repos: |
-      microsoft/mu_oem_sample
-
-# Azure Pipelines - Matrix Dependent - All
+# Azure Pipelines - Matrix Dependent - CLANDPDB

```

Comment:
We discussed dropping the `CLANGPDB` files. In case they are not.


```suggestion
# Azure Pipelines - Matrix Dependent - CLANGPDB
```

Reply:
Will remove shortly

Reply:
These steps are removed.